### ISSUE TITLE
CLI downgrade command raises exception

### ISSUE THREAD

**Issue by tadams42** [davidism/flask-alembic#8]
When calling `db downgrade` without any arguments from my Flask app, I get following:

```traceback
  File "my_project/.venv/lib/python3.11/site-packages/flask_alembic/cli.py", line 118, in downgrade
    alembic.downgrade(target)
  File "my_project/.venv/lib/python3.11/site-packages/flask_alembic/extension.py", line 408, in downgrade
    self.run_migrations(do_downgrade)
  File "my_project/.venv/lib/python3.11/site-packages/flask_alembic/extension.py", line 272, in run_migrations
    env.run_migrations(**kwargs)
  File "my_project/.venv/lib/python3.11/site-packages/alembic/runtime/environment.py", line 948, in run_migrations
    self.get_context().run_migrations(**kw)
  File "my_project/.venv/lib/python3.11/site-packages/alembic/runtime/migration.py", line 615, in run_migrations
    for step in self._migrations_fn(heads, self):
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "my_project/.venv/lib/python3.11/site-packages/flask_alembic/extension.py", line 403, in do_downgrade
    return self.script_directory._downgrade_revs(  # type: ignore[return-value]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "my_project/.venv/lib/python3.11/site-packages/alembic/script/base.py", line 473, in _downgrade_revs
    return [
           ^
  File "my_project/.venv/lib/python3.11/site-packages/alembic/script/base.py", line 473, in <listcomp>
    return [
           ^
  File "my_project/.venv/lib/python3.11/site-packages/alembic/script/revision.py", line 813, in iterate_revisions
    revisions, heads = fn(
                       ^^^
  File "my_project/.venv/lib/python3.11/site-packages/alembic/script/revision.py", line 1330, in _collect_downgrade_revisions
    branch_label, target_revision = self._parse_downgrade_target(
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "my_project/.venv/lib/python3.11/site-packages/alembic/script/revision.py", line 1112, in _parse_downgrade_target
    assert isinstance(
AssertionError: Expected downgrade target in string form
```

I did some debugging to ensure that `alembic.downgrade(target)` is called with default argument and it looks like this:

```python
alembic.downgrade("-1")
```

problem happens here:

```python
# flask-alembic/extension.py
def downgrade(self, target: int | str | Script = -1) -> None:
    """Run migrations to downgrade database.

    :param target: Revision to go down to.
    """
    target_arg = self._simplify_rev(target, handle_int=True)

    def do_downgrade(
        revision: str | list[str] | tuple[str, ...], context: MigrationContext
    ) -> list[MigrationStep]:
        return self.script_directory._downgrade_revs(  # type: ignore[return-value]
            target_arg,  # type: ignore[arg-type]
            revision,  # type: ignore[arg-type]
        )
```

`self._simplify_rev()` always returns list of strings, but notice that method `_downgrade_revs` in `alembic` is declared as following:

```python
# alembic/script/base.py
def _downgrade_revs(
    self, destination: str, current_rev: Optional[str]
) -> List[RevisionStep]:
    ...
```

No time to dig deeper, hope that above helps.

**Comment by davidism** [comment]
Fixed and available as 3.0.1 on PyPI.

**Comment by tadams42** [comment]
Thnx!
